_includes/home/hero.html

```diff
@@ -4,7 +4,7 @@
     <h1 class="hero-heading">Open source software for<br> developing world hospitals.</h1>
 
     <div class="cta-row">
-      <a href="/demo" class="cta primary">View demo</a>
+      <a href="/tryit" class="cta primary">Try it Now!</a>
```

Comment:
Same here with punctuation. Also, I think the "Now" makes it feel too cheesy/markety. How about just: `Try the demo`